### PR TITLE
contributing: Remove parts of locale readme already present in submitting guidelines

### PR DESCRIPTION
**Pull request by marisn**
Most of locale README content was already being present in submission guidelines imported from the trac.

### CODE REVIEW

File: doc/development/submitting/python.md [hunk 1]
```diff
@@ -301,7 +301,7 @@ output. This is reserved for standard module output if it has one.
 ### Translations
 
 To enable translating of messages to other languages (than English), use full
-strings, e.g.
+strings, e.g. (good):
```
Comment (neteler) [inline doc/development/submitting/python.md:304]:
```suggestion
strings, e.g. (good example):
```

File: doc/development/submitting/submitting_c.md [hunk 2]
```diff
@@ -261,9 +261,11 @@ sent to any of these functions will be printed to stderr.
 
 G_message() output is not expected to be sent to pipe or file.
 
-Always use the gettext macros with _("") for user messages, example:
+Always use the gettext macros with \_("") for user messages, example:
 
 ```c
+#include "glocale.h"
+
```
Comment (wenzeslaus) [inline doc/development/submitting/submitting_c.md:268]: Ideally, this should be a separate code block with its own description.

File: doc/development/submitting/submitting_c.md [hunk 3]
```diff
@@ -261,9 +261,11 @@ sent to any of these functions will be printed to stderr.
...
 G_fatal_error(_("Vector map <%s> not found"), name);
 ```
```
Comment (wenzeslaus) [inline doc/development/submitting/submitting_c.md:271]: GTC: Here the examples are using "GTC" only. The code has a mixture of "GTC" and "GTC:". I see "GTC:" in your new code. Do you want to switch to that here too? "GTC:" seems like a better practice to me.

Comment (marisn) [reply doc/development/submitting/submitting_c.md:271]: Good catch! I looked into it and it does not matter for comment extraction. Comments are extracted to the POT file as a whole line thus we can freely decide if it should be "GTC" or "GTC:"

Comment (marisn) [reply doc/development/submitting/submitting_c.md:277]: I changed the text to the "GTC:"

File: doc/development/submitting/submitting_c.md [hunk 4]
```diff
@@ -261,32 +261,53 @@ sent to any of these functions will be printed to stderr.
...
+Afterwards mark all user visible strings with the gettext macro \_("message"):
 
 ```c
 G_fatal_error(_("Vector map <%s> not found"), name);
 ```
 
 It is suggested to add a comment line before translatable user message to give a
 hint to translators about meaning or use of cumbersome or obscure message. First
-word in the comment must be GTC - GRASS translation comment,
+word in the comment must be GTC: - GRASS translation comment,
```
Comment (veroandreo) [inline doc/development/submitting/submitting_c.md:280]:
```suggestion
word in the comment must be GTC: GRASS translation comment,
```

File: doc/development/submitting/submitting_c.md [hunk 5]
```diff
@@ -261,32 +261,53 @@ sent to any of these functions will be printed to stderr.
...
 
 Example:
 
 ```c
-/* GTC A name of a projection */
+/* GTC: A name of a projection */
```
Comment (veroandreo) [inline doc/development/submitting/submitting_c.md:285]:
```suggestion
/* GTC: Name of a projection */
```

File: doc/development/submitting/submitting_c.md [hunk 6]
```diff
@@ -261,32 +261,53 @@ sent to any of these functions will be printed to stderr.
...
 G_message(_("State Plane"));
 ```
 
 Any message with a noun in plural form has to pass `n_()` macro, even if for the
-English language it is not required!
+English language it is not required! The syntax is
```
Comment (veroandreo) [inline doc/development/submitting/submitting_c.md:290]:
```suggestion
English language is not required! The syntax is
```

File: locale/README.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+If you have any difficulties please ask for help
+on the GRASS development mailing list `grass-dev lists.osgeo.org`
+(subscribe at <https://lists.osgeo.org/mailman/listinfo/grass-dev>). Your
+willingness, time, and effort translating are valuable.
+
+Note that GRASS must be configured with `--with-nls` and (re)compiled
+to make use of the translated messages.
+
+PLEASE HELP TO TRANSLATE GRASS MESSAGES!
```
Comment (veroandreo) [inline locale/README.md:17]:
```suggestion
**Please help us translate all the GRASS messages!!**
```

Comment (veroandreo) [reply locale/README.md:17]: Caps are usually interpreted as shouting, to highlight text I suggest to use bold face.

**Review by veroandreo — COMMENTED** [review]
minor edits suggested

**Pull request merged**